string home_folder
string current_folder
string selected_folder
string[] current_folders
string[] supported_file_types
int32 file_count
string current_file
string current_fps

bool paused

string[] size_options_list
string set_size
string[] encoding_options_list
string set_encoding


bool set_random
bool set_overlay


bool running

